Realm/ObjectServerTests/SwiftSyncTestCase.swift

```diff
@@ -74,7 +63,7 @@ class SwiftSyncTestCase: RLMSyncTestCase {
     }
 
     func synchronouslyLogInUser(for credentials: Credentials,
-                                file: StaticString = #file,
+                                file: StaticString = #filePath,
```

Comment:
This breaks Xcode 11. The Swift change was poorly thought out and there's no reasonable way to support both 11 and 12 without warnings, so they're thinking of changing things in a future Xcode beta.